src/Fantomas/CodePrinter.fs

```diff
@@ -1250,6 +1250,7 @@ and genExpr astContext synExpr ctx =
                 genTriviaFor SynExpr_YieldOrReturnFrom_ReturnBang returnBangKeyword !- "return! "
             | Do doKeyword -> genTriviaFor SynExpr_Do_Do doKeyword !- "do "
             | DoBang doBangKeyword -> genTriviaFor SynExpr_DoBang_DoBang doBangKeyword !- "do! "
+            | Fixed fixedKeyword -> genTriviaFor SynExpr_Fixed fixedKeyword !- "fixed "
```

Comment:
This line surprised me a little bit.
You already had `SynExpr_Fixed` but it was not supported in `SingleExp` - strange!

Reply:
Thank for so detailed explanation! I hope that I did it right =)